### PR TITLE
[USER32_APITEST] Improve GetMessageTime tests more

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Compare `GetMessageTime` and `GetTickCount` values.
JIRA issue: [CORE-15565](https://jira.reactos.org/browse/CORE-15565)

### CODE REVIEW

**Comment by katahiromz** [comment]
Win10:
user32_apitest-win10-after
Win2k3:
user32_apitest-win2k3
ReactOS:
user32_apitest-ros-after

**Comment by katahiromz** [comment]
MsqCalculateMessageTime seems like wrong...
It must return a 32-bit tick count value.

File: modules/rostests/apitests/user32/GetMessageTime.c [hunk 1]
```diff
@@ -47,6 +47,8 @@ WindowProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
             {
                 ok(GetMessageTime() - s_nMsgTime >= TIMER_INTERVAL / 2,
                    "message time is wrong\n");
+                ok(GetTickCount() - (DWORD)GetMessageTime() < TIMER_INTERVAL / 2,
+                   "message time is wrong\n");
```
Comment (SergeGautherie) [inline modules/rostests/apitests/user32/GetMessageTime.c]: May I suggest "GetMessageTime() is wrong, compared to GetTickCount()\n", to be more explicit? Everywhere.

Comment (katahiromz) [reply modules/rostests/apitests/user32/GetMessageTime.c]: Thanks!

**Pull request merged**